app/src/main/java/cm/aptoide/pt/promotions/PromotionsHomeDialog.java

```diff
@@ -35,6 +38,11 @@ public PromotionsHomeDialog(Context context) {
     dialog.setCancelable(true);
     dialog.setCanceledOnTouchOutside(true);
 
+    Window window = dialog.getWindow();
+    if (window != null) {
+      window.setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
+    }
```

Comment:
Is this there reason why the dialog is not showing the grey background?

Reply:
It isn't, this is here because of another background 'glitch'. This also happens in dev, which means it couldn't be because of this.